service/history/workflowTaskHandler.go

```diff
@@ -96,12 +97,13 @@ func newWorkflowTaskHandler(
 		namespaceEntry:          namespaceEntry,
 
 		// internal state
-		hasBufferedEvents:           mutableState.HasBufferedEvents(),
-		failWorkflowTaskInfo:        nil,
-		activityNotStartedCancelled: false,
-		continueAsNewBuilder:        nil,
-		stopProcessing:              false,
-		mutableState:                mutableState,
+		hasBufferedEvents:                 mutableState.HasBufferedEvents(),
+		failWorkflowTaskInfo:              nil,
+		activityNotStartedCancelled:       false,
+		continueAsNewBuilder:              nil,
+		stopProcessing:                    false,
+		mutableState:                      mutableState,
+		initiatedChildExecutionsInSession: make(map[string]struct{}),
```

Comment:
Do we have a similar problem with other commands? What about activities, timers, etc?

Reply:
I will add integration tests for timer and activity use cases in a separate PR.